src/createAnimatedComponent.tsx

```diff
@@ -139,6 +140,7 @@ function extractSharedValuesMapFromProps(
     if (key === 'style') {
       const styles = flattenArray<StyleProps>(props.style ?? []);
       styles.forEach((style) => {
+        if (!style) return
```

Comment:
We should also protect when key !== "style", for example we crashed line L154 with testID={null}

Reply:
I think this case is covered by https://github.com/software-mansion/react-native-reanimated/pull/4128/files